public/styles/_home.scss

```diff
@@ -24,176 +24,56 @@
   max-height: 200px;
 }
 
-// Companies section --------------------------
+// Featured apps section --------------------------
 
-.featured-company-list-item {
-  flex-basis: 250px;
-  height: 104px;
-
-  img {
-    width: 250px;
-    height: 104px;
-  }
-}
-
-// Apps section ------------------------------
-
-/* Featured Apps with Icons */
-
-.featured-app {
-  display: inline-block !important;
-  vertical-align: top !important;
-  width: 140px !important;
-  padding: 0 !important;
-  color: $main-color-subtle !important;
-  margin: 10px 20px !important;
-  &:hover {
-    text-decoration: none !important;
-  }
-}
-
-.featured-app-logo-wrapper {
-  width: 120px !important;
-  height: 120px !important;
-  margin: 10px !important;
-  transition: transform .32s !important;
-}
-
-.featured-app:hover .featured-app-logo-wrapper {
-  transform: scale(1.1);
-  transition-duration: .08s;
-}
-
-.featured-app:active .featured-app-logo-wrapper {
-  transform: scale(.92);
-  transition-duration: 0;
-}
-
-.featured-app-logo {
-  width: 100%;
-}
-
-.featured-app-name {
-  text-align: center;
-  text-decoration: none;
-  line-height: 1.3;
-  font-weight: 300;
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
-}
-
-/* Listed Apps with Icons */
-
-$app-list-icon-size: 48px;
-
-.app-list {
-  list-style: none;
-  margin: 0;
-  padding: 0;
-}
-
-.listed-app {
-  text-align: left;
-}
-
-// Override .filterable-list
-.listed-app.listed-app {
-  padding: 0;
-  margin-bottom: 0;
-  border-bottom: none;
-}
-
-.listed-app > a {
+.featured-apps {
   display: flex;
-  padding: 10px;
-  border: 1px solid transparent;
-}
-
-.listed-app > a:hover,
-.listed-app > a:focus {
-  text-decoration: none;
-  text-decoration: none;
-  border-color: $main-border-color;
-  background-color: $main-bg-color-shade;
-  outline: none;
+  justify-content: center;
+  max-width: 90%;
+  margin-left: auto;
+  margin-right: auto;
 }
 
-.listed-app-logo-wrapper {
+a.featured-app-list-item {
+  background-color: #e4e4e4;
```

Comment:
Can we do some flex magic on this class to make the boxes have the correct width before the images load.  On my machine when I load the page I can see the images load in asynchronously and it causes this strange stutter effect because these boxes don't have fixed widths but they are based on the images themselves.

Hard to capture a gif but if you scroll down and reload while looking at the images you should see what I mean, kinda a nit but should be easy enough to solve with a `flex: 1` or something